sklearn/feature_extraction/text.py

```diff
@@ -1342,6 +1342,34 @@ class TfidfTransformer(TransformerMixin, BaseEstimator):
         The inverse document frequency (IDF) vector; only defined
         if  ``use_idf`` is True.
 
+    Examples
+    --------
+    >>> from sklearn.feature_extraction.text import TfidfTransformer
```

Comment:
We can use a pipeline to with `CountVectorizer` to create the count matrix with a custom vocabulary:

```py
from sklearn.feature_extraction.text import TfidfTransformer
from sklearn.feature_extraction.text import CountVectorizer
from sklearn.pipeline import Pipeline
import numpy as np
corpus = ['this is the first document',
          'this document is the second document',
          'and this is the third one',
          'is this the first document']
vocabulary = ['this', 'document', 'first', 'is', 'second', 'the',
              'and', 'one']
pipe = Pipeline([('count', CountVectorizer(vocabulary=vocabulary)), 
                  ('tfid', TfidfTransformer())])
pipe.fit(corpus)
pipe['count'].transform(corpus).toarray()
pipe['tfid'].idf_
pipe.transform(corpus).shape
```

Reply:
Yeah, sure! I'll work on this.

Reply:
I think it's done!